engine/src/main/java/com/sandboni/core/engine/sta/connector/ReflectionCallConnector.java

```diff
@@ -0,0 +1,36 @@
+package com.sandboni.core.engine.sta.connector;
+
+import com.sandboni.core.engine.sta.Context;
+import com.sandboni.core.engine.sta.graph.Link;
+import com.sandboni.core.engine.sta.graph.LinkFactory;
+import com.sandboni.core.engine.sta.graph.LinkType;
+import com.sandboni.core.engine.sta.graph.vertex.Vertex;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static com.sandboni.core.engine.sta.graph.vertex.VertexInitTypes.REFLECTION_CALL_VERTEX;
+
+/**
+ * Connects between a test marked with LinkType.REFLECTION_CALL_TEST to all it's tests.
+ */
+public class ReflectionCallConnector implements Connector {
+
+    @Override
+    public void connect(Context context) {
```

Comment:
why we need it? can't we found all REFLECTION_CALL_TEST to dummy REFLECTION node links in order to get the tests?

Reply:
REFLECTION_CALL_TEST will only contain the test class name. we need all the test cases from this class.

Reply:
But if you use the method name in the link, as my above comment, then you will have the test case, right?

Reply:
You are right. changed